ansible_collections/arista/avd/plugins/plugin_utils/eos_designs_facts.py

```diff
@@ -126,6 +133,21 @@ def default_interfaces(self):
 
         return {}
 
+    @cached_property
+    def default_node_type(self):
```

Comment:
please add an underscore in front of the name `_default_node_type`, to avoid it being part of the output facts. We don't need this data outside of this class.
Also adjust docstring

Reply:
Thanks, will fix.